### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by kloczek** [nayuki/QR-Code-generator#83]
I think that it would be good to flush currently committed changes and make new release :)

Looks like already at least few distributions are no longer using regular 1.5.0.

**Comment by nayuki** [comment]
Thanks, that's a good idea. I hesitated because I didn't make functional changes since v1.5.0. But it looks like I did a lot of work since then. Upon reviewing my series of commits, these are the main things I did:

* Clarified a lot of internal code (especially for Rust, C++, and C).
* Merged the C++ source files from 3 pairs to just 1 pair.
* Modified internal assertions (which should not trigger unless I made a mistake).
* Deleted JavaScript in favor of TypeScript.
* Made very minimal changes to Java and Python source codes.

Given the same inputs, the libraries as of latest (commit 13a25580a3e2a29b2b6653802d58d39056f3eaf2) still generate the exact same output as v1.5.0. Also, I keep master stable, so any project that took a non-tagged commit from my project is okay and not risky.
